src/Form/RedirectForm.php

```diff
@@ -76,7 +76,7 @@ public function form(array $form, FormStateInterface $form_state) {
         '#type' => 'language_select',
         '#title' => t('Language'),
         '#languages' => Language::STATE_ALL,
```

Comment:
You can enable that at admin/config/regional/content-language. We possibly want to provide default configuration for that, as it makes a lot more sense to do it here than for most other entity types. Should work by configuring it and then putting the language.content_settings.redirect.redirect.yml config entity into config/optional.